.copier-answers.yml

```diff
@@ -0,0 +1,14 @@
+# Changes here will be overwritten by Copier
+_commit: v0.3.0
+_src_path: gl:climate-resource/copier-core-python-repository
+email: zebedee.nicholls@climate-energy-college.org
+initial_setup: true
```

Comment:
Do I misunderstand `initial_setup`? Shouldn't it be `false` because the repo existed in non-copier form previously?

Reply:
I don't know how important this is for copiers internal stuff, maybe it doesn't matter.

Reply:
Fixed now.